external/webL10n/l10n.js

```diff
@@ -1,5 +1,4 @@
-/**
- * Copyright (c) 2011-2013 Fabien Cazenave, Mozilla.
+/* Copyright 2014 Mozilla Foundation
```

Comment:
Please revert this change; and also the rest of the changes of Copyright year in the rest of this patch.